### PR TITLE
data-quality-report - Various

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Previously we had quality checks in 2 places and all checks appeared in 1 place only - DQR or import. Now they are all in DQR - and DQR shows up on import - so all checks appear in both places!

File: indigo/dataqualityreport.py [hunk 1]
```diff
@@ -99,6 +118,81 @@ def get_count_errors_in_priority_levels(self):
...
+        # Look through source table looking for problems
+        for source_item in source_list:
+            source_id = jsonpointer.resolve_pointer(
+                source_item, TYPE_PROJECT_SOURCE_LIST["item_id_key"], default=""
+            )
+            if source_id:
+                source_id = str(source_id).strip()
+                source_ids_found.append(source_id)
+                # Is this source ID used? Add to list if not
+                if (
```
Comment (Lathrisk) [inline indigo/dataqualityreport.py:182]: Is there a reason this is formatted like this?

Comment (ghost) [reply indigo/dataqualityreport.py:182]: black?

Comment (Lathrisk) [reply indigo/dataqualityreport.py:182]: Yeah, I wondered that, some funky line splits.

**Review by Lathrisk — APPROVED** [review]

**Pull request merged**